From ceecad71b2892afd683b4b6dff1577bde95fdfa1 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 30 Jun 2017 15:11:03 +0800
Subject: [PATCH 2031/5242] MLK-15314 driver: clk: Change the audio ahb clock
 to sys2_pll_500m

commit  a2dd5084132c83cf85bc3ac419137b26ccfee690 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change the audio ahb clock source to sys2_pll_500m clock.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8mq.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/clk/imx/clk-imx8mq.c b/drivers/clk/imx/clk-imx8mq.c
index 9e854f8..5d471db 100644
--- a/drivers/clk/imx/clk-imx8mq.c
+++ b/drivers/clk/imx/clk-imx8mq.c
@@ -803,8 +803,7 @@ static void __init imx8mq_clocks_init(struct device_node *ccm_node)
 		clk_prepare_enable(clks[clks_init_on[i]]);
 
 	clk_set_parent(clks[IMX8MQ_CLK_AHB_SRC], clks[IMX8MQ_SYS1_PLL_133M]);
-	clk_set_parent(clks[IMX8MQ_CLK_AUDIO_AHB_SRC], clks[IMX8MQ_SYS2_PLL_1000M]);
-	clk_set_rate(clks[IMX8MQ_CLK_AUDIO_AHB_DIV], 500000000);
+	clk_set_parent(clks[IMX8MQ_CLK_AUDIO_AHB_SRC], clks[IMX8MQ_SYS2_PLL_500M]);
 
 	/* increase NOC clock to achieve best DDR access performance */
 	clk_set_rate(clks[IMX8MQ_CLK_NOC_DIV], clk_get_rate(clks[IMX8MQ_SYS1_PLL_800M]));
-- 
1.7.9.5

